kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/group/GroupCoordinator.java

```diff
@@ -555,20 +558,24 @@ private void doSyncGroup(GroupMetadata group,
 
     public CompletableFuture<Errors> handleLeaveGroup(
         String groupId,
-        String memberId
+        Set<String> members
     ) {
         return validateGroupStatus(groupId, ApiKeys.LEAVE_GROUP).map(CompletableFuture::completedFuture
         ).orElseGet(() -> groupManager.getGroup(groupId).map(group -> group.inLock(() -> {
-            if (group.is(Dead) || !group.has(memberId)) {
+            if (group.is(Dead)) {
+                return CompletableFuture.completedFuture(Errors.UNKNOWN_MEMBER_ID);
```

Comment:
The error should be `Errors.COORDINATOR_NOT_AVAILABLE`?

Reply:
Sure, I just found the Kafka 2.8 code is using `Errors.COORDINATOR_NOT_AVAILABLE`. We can change it in next PR if necessary.

https://github.com/apache/kafka/blob/2.8/core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala#L470-L471

Reply:
let me follow up with another patch on this

Reply:
@Demogorgon314 finally I updated this value in this PR